e2e/scaling/scaling_test.go

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package scaling
+
+import (
+	"testing"
+	"time"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/hashicorp/nomad/e2e/e2eutil"
+	"github.com/hashicorp/nomad/helper/pointer"
+	"github.com/hashicorp/nomad/helper/uuid"
+	"github.com/hashicorp/nomad/nomad/structs"
+	"github.com/shoenig/test/must"
+	"github.com/shoenig/test/wait"
+)
+
+const defaultNS = "default"
+
+func TestScaling(t *testing.T) {
+
+	nomad := e2eutil.NomadClient(t)
+	e2eutil.WaitForLeader(t, nomad)
+	e2eutil.WaitForNodesReady(t, nomad, 1)
```

Comment:
if we're switching to new test format, and these tests expect linux nodes in particular, could ensure that directly here with the `v3` e2e package, e.g.

https://github.com/hashicorp/nomad/blob/a9d61ea/e2e/example/example_test.go#L19-L23

```go
func TestExample(t *testing.T) {
	cluster3.Establish(t,
		cluster3.Leader(),
		cluster3.LinuxClients(1),
		cluster3.Timeout(3*time.Second),
	)
```

but realistically, I don't expect that a windows client will ever win the race over linux to start.